plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -120,6 +124,75 @@ def __init__(self, module, connection_header):
         self.validate_certs = self.module.params.get('validate_certs')
         self.restheaders = connection_header
 
+    def get_realm_by_id(self, realm='master'):
+        """ Obtain realm representation by id
+
+        :param realm: realm id
+        :return: dict of real, representation or None if none matching exist
+        """
+        realm_url = URL_REALM.format(url=self.baseurl, realm=realm)
+
+        try:
+            return json.loads(to_native(open_url(realm_url, method='GET', headers=self.restheaders,
+                                                 validate_certs=self.validate_certs).read()))
+
+        except HTTPError as e:
+            if e.code == 404:
+                return None
+            else:
+                self.module.fail_json(msg='Could not obtain realm %s: %s'
+                                          % (realm, str(e)))
+        except ValueError as e:
+            self.module.fail_json(msg='API returned incorrect JSON when trying to obtain realm %s: %s'
+                                      % (realm, str(e)))
+        except Exception as e:
+            self.module.fail_json(msg='Could not obtain realm %s: %s' % (realm, str(e)),
+                                      exception=traceback.format_exc())
```

Comment:
```suggestion
                                  exception=traceback.format_exc())
```
I think @russoz means this indent, and similar ones. I agree that they look misleading, since `exception` is not a part of the expression for `msg`.

Reply:
Yes, that one - it seems it was added like that in multiple places. Thanks for helping pointing it out, Felix.

Reply:
Ok I fixed indentation and add missing traceback